### PR TITLE
Hotfix/sg 000 reduce import loops risk

### PR DESCRIPTION
**Pull request by Louis-Dupont**
PR to prevent import loops that blocked me on the DeciClient PR. It's basically just using full import path, and moving one function from training to common:

### CODE REVIEW

File: src/super_gradients/common/aws_connection/aws_secrets_manager_connector.py [hunk 1]
```diff
@@ -95,18 +96,26 @@ def __get_secrets_manager_dict_for_secret_name(aws_env: str, secret_name: str) -
...
             logger.debug(f'Fetching the secret "{secret_name}" in env "{aws_env}"')
             aws_secrets = AWSSecretsManagerConnector.current_environment_client.get_secret_value(SecretId=secrets_path)
-            aws_secrets_dict = json.loads(aws_secrets['SecretString'])
+            aws_secrets_dict = json.loads(aws_secrets["SecretString"])
             return aws_secrets_dict
 
         except Exception as ex:
-            error = '[' + current_class_name + '] - Caught Exception while trying to connect to aws to get credentials from secrets manager: ' + '"' + str(
-                ex) + '"' + ' for ' + str(secrets_path)
+            error = (
```
Comment (ofrimasad) [inline src/super_gradients/common/aws_connection/aws_secrets_manager_connector.py:109]: I know it's just some black formatter automatic thing, but can you please fix this ugly ++++ and replace it with a formatted string?

File: src/super_gradients/common/data_connection/s3_connector.py [hunk 2]
```diff
@@ -202,22 +189,21 @@ def download_key(self, target_path: str, key_to_download: str) -> bool:
...
+            self._logger.debug("Uploading Key: " + key_to_download + " from S3 bucket: " + self.bucket_name)
             self.s3_client.download_file(Bucket=self.bucket_name, Filename=target_path, Key=key_to_download)
         except botocore.exceptions.ClientError as ex:
-            if ex.response['Error']['Code'] == '404':
-                self._logger.error('[' + sys._getframe().f_code.co_name + '] - Key does exist in bucket)')
+            if ex.response["Error"]["Code"] == "404":
+                self._logger.error("[" + sys._getframe().f_code.co_name + "] - Key does exist in bucket)")
             else:
                 self._logger.critical(
-                    '[' + sys._getframe().f_code.co_name + '] - Caught Exception while trying to download key ' + str(
```
Comment (ofrimasad) [inline src/super_gradients/common/data_connection/s3_connector.py:212]: here also please

**Review by ofrimasad — COMMENTED** [review]
just two minor comments

File: src/super_gradients/common/aws_connection/aws_secrets_manager_connector.py (hunk 1 above)
Comment (Louis-Dupont) [reply src/super_gradients/common/aws_connection/aws_secrets_manager_connector.py:109]: Fixed in 20ac8a290ce023b49df1cb9d67eb77c663a792d1

File: src/super_gradients/common/data_connection/s3_connector.py (hunk 2 above)
Comment (Louis-Dupont) [reply src/super_gradients/common/data_connection/s3_connector.py:212]: Fixed in 20ac8a290ce023b49df1cb9d67eb77c663a792d1

**Review by ofrimasad — APPROVED** [review]
LGTM

**Pull request merged**